flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/functions/casting/CastRuleProvider.java

```diff
@@ -203,6 +224,7 @@ private CastRuleProvider addRule(CastRule<?, ?> rule) {
             /* lookup by any family matching */
             for (LogicalTypeFamily family : type.getFamilies()) {
                 out = map.get(family);
+                // TODO what if multiple?
```

Comment:
we should avoid TODOs to the code base, either we fix it or it is not an issue currently